core/modules/filter/filter.api.php

```diff
@@ -134,28 +134,32 @@ function hook_filter_info_alter(&$info) {
  *   which should be unique and therefore prefixed with the name of the module.
  *   Each value is an associative array describing the editor, with the
  *   following elements (all are optional except as noted):
- *   - title: (required) A human readable name for the editor.
+ *   - label: (required) A human readable name for the editor.
  *   - settings callback: The name of a function that returns configuration
  *     form elements for the editor. See hook_editor_EDITOR_settings() for
  *     details.
  *   - default settings: An associative array containing default settings for
  *     the editor, to be applied when the editor has not been configured yet.
+ *   - file: The name of a file containing the editor settings callback.
+ *   - library: An associative array containing additional libraries.
  *   - js settings callback: The name of a function that returns configuration
```

Comment:
Not sure, but I think it can only be one library, keyed by the module name.

Reply:
Ok, I will change the wording. However, looking at CKEditor's implementation I believe this should still be an array.